### ISSUE TITLE
Could we add possibility to assign field as invocation appropriate setter.

### ISSUE THREAD

**Issue by fAns1k** [techery/janet-http#14]
as an example `kotlin` doesn't have fields at all and uses `KPropery<*>` instead of. in this case we cannot assign value in the generated code. 

Resolution: 
complier could check whether filed exist or not to solve the assignment strategy. (field assigment or invoke corresponding setter).

**Comment by almozavr** [comment]
We can, and it's a good practice. The thing is we'd better use some tool to follow strategy of field assignment. Imagine All those Immutables, AutoValue, etc.

**Comment by fAns1k** [comment]
@almozavr yep, to avoid boiler-plate like `setParam()` for each parameter.

**Comment by almozavr** [comment]
@fAns1k partially fixed with a88a78b04eee56a411bf1cb0d1fcac9d151cefe3 (kotlin action classes support)
